src/modules/extendedCallWindow/assets/vehicleListPermanentSearch.ts

```diff
@@ -0,0 +1,22 @@
+export default (): void => {
+    const vehicleTabs = document.querySelector('#tabs');
+    if (vehicleTabs) vehicleTabs.addEventListener('click', vehicleListChange);
+    //Initial expanding
+    vehicleListChange()
+
+    /**
+     * Gets called on every vehicle tab change to update the vehicle search bar
+     * (for some reasons they have different search bars instead of filters).
+     */
+    function vehicleListChange() {
+        const vehicleSearch = document
+            .querySelector('#vehicle_list_step>.tab-content>.active')
+            ?.querySelector(
+                "[id^='mission_vehicle_search_outer_']"
+            ) as HTMLInputElement;
+        const enableSearchBtn = document
+            .querySelector('#vehicle_list_step>.tab-content>.active')
+            ?.querySelector("[id^='show_search_']") as HTMLAnchorElement;
```

Comment:
I would simplify this querySelector too

Reply:
Will return in a different result
![image](https://github.com/LSS-Manager/LSSM-V.4/assets/69073372/e3b94f7b-4c8f-4325-a099-0e243a06790e)

Reply:
you've added a `>` after the `.active` in the combined selector. That requires the element that matches `[id^='show_search_']` to be a direct sibling of `.active`. Without the `>` it should work correctly.